Vladyslav-Hudz/README.md

```diff
@@ -1,4 +1,4 @@
-# Arduino - Get Started Guide
+# Arduino - get started guide
 
 This document will provide you with the initial instructions for using Arduino and getting started with the platform.
 
```

Comment:
This comment is for "Read an [introduction](https://www.arduino.cc/en/Guide/Introduction) on what is Arduino and why you'd want to use it."
The word order should be slightly different --> ...on what Arduino **is**...